### DISCUSSION TITLE
CPUExecutionProvider and CUDAExecutionProvider performance

### DISCUSSION THREAD

**Discussion by trungpham2606** [zhiqwang/yolov5-rt-stack#160] [Q&A] [answered]
Hello @zhiqwang 
After exporting successfully yolov5x (+nms) to onnx model.
I did the test on my dataset used both CPUExecutionProvider and CUDAExecutionProvider.
With CUDAExecutionProvider, the speed is 10 times faster than CPUExecutionProvider  but the results are very bad and not stable.
How can I deal with this problem ?

**Comment by zhiqwang** [comment]
Hi @trungpham2606 , Thank you for your feedback on this phenomenon here.

Since I haven't resolved issue #159 yet, I cannot reproduce your problem and I'm not sure why this is happening here. I think that you can first test if there is any difference in the inference results of this [notebook](https://github.com/zhiqwang/yolov5-rt-stack/blob/master/notebooks/export-onnx-inference-onnxruntime.ipynb) on the CPU and GPU devices.

**Comment by trungpham2606** [comment, marked as answer]
Hi @zhiqwang 
It's just about the onnxruntime-gpu version. Update it works well for me ^^
